assets/prometheus-operator-user-workload/0probe-custom-resource-definition.yaml

```diff
@@ -0,0 +1,212 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Why do we need the CRD in this directory?

Reply:
fixed (i think this was an intermediate state)